plugins/postgres/dbt/include/postgres/macros/materializations/materialized_view.sql

```diff
@@ -0,0 +1,84 @@
+{% macro postgres__get_alter_materialized_view_as_sql(
+    relation,
+    configuration_changes,
+    sql,
+    existing_relation,
+    backup_relation,
+    intermediate_relation
+) %}
+
+    -- apply a full refresh immediately if needed
+    {% if configuration_changes.requires_full_refresh %}
+
+        {{ get_replace_materialized_view_as_sql(relation, sql, existing_relation, backup_relation, intermediate_relation) }}
+
+    -- otherwise apply individual changes as needed
+    {% else %}
+
+        {{ postgres__update_indexes_on_materialized_view(relation, configuration_changes.indexes) }}
```

Comment:
idea here being - additional updates to other configuration can occur sequentially in this `else` block?

e.g: 
```
{{ postgres__update_indexes_on_materialized_view(relation, configuration_changes.indexes) }}
{{ postgres__update_<something-else>_on_materialized_view(relation, configuration_changes.something_else) }}
```

Reply:
That's the idea. It would be one or more changes.